.gitignore

```diff
@@ -26,3 +26,8 @@ telepresence.log
 
 # distutils gunk
 leastauthority.com.egg-info/
+
+# env file
+.env
```

Comment:
Is it expected that folks will have a `.env` file around?  Is this where `env-cmd` takes its values from?  Or is there another tool that's reading this file to initialize environment variables for local/development deployments of the backend?